bot/helper/constant.ts

```diff
@@ -2,3 +2,8 @@ export class DefaultLineSetting {
   public static readonly defaultStartLine = 0;
   public static readonly defaultEndLine = 9;
 }
+
+export class DefaultCodeStyle {
+  public static readonly vsCSSStyle =
```

Comment:
CSS(Cascading Style Sheets) style sounds weird. May be `vsStyleInCSS` or just `vsCSS`?

Reply:
And I prefer the style string only contains `<style>***</style>`. We can have another wrap method:
```
function wrapRawCodeWithCSS(code, css) {
    return "<span> + css + "<pre><code>" + code + "</code></pre></span>"
}
```